RUFAS/biophysical/field/soil/soil_data.py

```
@@ -1,7 +1,7 @@
 from copy import deepcopy
 from dataclasses import InitVar, dataclass, field
 from math import inf
-from typing import List, Optional
+from typing import List, Any
```

Ruff (0.15.15)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)